syz-fuzzer/fuzzer.go

```diff
@@ -629,6 +629,21 @@ func (fuzzer *Fuzzer) checkNewCallSignal(p *prog.Prog, info *ipc.CallInfo, call
 	return true
 }
 
+// nolint: unused
+// It's only needed for debugging.
+func (fuzzer *Fuzzer) logMessage(level int, msg string, args ...interface{}) {
```

Comment:
I would call it Logf, all logging logs messages, so that part is excessive and linter will check format string for Logf:
https://github.com/google/syzkaller/blob/master/tools/syz-linter/linter.go#L295

Reply:
Renamed the method. Or did you mean also something else?